### ISSUE TITLE
SagePay\Form redirect

### ISSUE THREAD

**Issue by dan-pascu** [thephpleague/omnipay-sagepay#144]
Hi,

I'm trying to use Laravel 6 with SagePay\Form and, as per the documentation, I have the following:

$gateway = OmniPay::create('SagePay\Form')->initialize([
    'vendor' => 'my_vendor',
    'testMode' => true,
    'encryptionKey' => 'my_encryption_key',
]);

$response = $gateway->authorize([
     'amount' => '9.99',
     'currency' => 'GBP',
     'transactionId' => 12345,,
     'returnUrl' => 'https://example.com/success',
     'failureUrl' => 'https://example.com/failure',
]);

$response->redirect();

And I get this error _Call to undefined method Omnipay\SagePay\Message\Form\AuthorizeRequest::redirect()_ 
When I dd the reponse I have this:
![image](https://user-images.githubusercontent.com/8133463/67303782-d2f05080-f4ea-11e9-8859-ebc505c361ca.png)

I think I'm missing something but I cannot figure out what, can you guide me please?

**Comment by judgej** [comment]
It looks like you have set `$response` to a request. To get the response, you need to "send" it. That doe snot actually send anything to Sage Pay, but it keeps the steps consistent across different gateways.

```php
$gateway = OmniPay::create('SagePay\Form')->initialize([...]);
$request = $gateway->authorize([...]);
$response = $request->send(); // The step you have have overlooked
$response->redirect();
```

**Comment by dan-pascu** [comment]
Following your instructions I manage to make it work. Thank you.

Previously I tried the send() method but it required a card, and for Form method I don't have a card to send. My mistake. Also in the card details are stored the billing and shipping information that are required and I didn't thought on this.

Another mistake I made is the encryptionKey. I included a wrong one and I received an encryption error: "openssl_encrypt(): IV passed is 20 bytes long which is longer than the 16 expected by selected cipher, truncating". I then used the correct Form Integration Encryption Password from SagePay and everything went fine.
